addon/ng2/blueprints/component/files/__path__/__name__.component.spec.ts

```diff
@@ -10,14 +10,28 @@ import {
   ComponentFixture,
   TestComponentBuilder
 } from 'angular2/testing';
-import {provide} from 'angular2/core';
+import {provide} from 'angular2/core';<% if (route) { %>
+import {Router, RouteParams} from 'angular2/router';<% } %>
 import {<%= classifiedModuleName %>Component} from './<%= dasherizedModuleName %>.component';
+<% if (route) { %>
+class MockRouter {
+  navigate() { }
+  registerPrimaryOutlet() { }
+}
 
-describe('<%= classifiedModuleName %> Component', () => {
-
-  beforeEachProviders((): any[] => []);
-
+class MockRouteParams {
```

Comment:
I think it will be quite common to use parameters in routes, and I'd prefer to leave this in.